training/trillium/GPT3-175B-MaxText/README.md

```diff
@@ -0,0 +1,30 @@
+# Instructions for training GPT3-175B-Maxtext on TPU trillium
+
+## XPK setup
+Please follow this [link](https://github.com/AI-Hypercomputer/tpu-recipes/training/trillium/XPK_README.md) to create your GKE cluster with XPK
+
+## Prep for Maxtext 
+Please follow this [link](https://github.com/AI-Hypercomputer/tpu-recipes/training/trillium/MAXTEXT_README.md) to install maxtext and build docker image
+
+## Run Maxtext GPT3-175B workloads on GKE
+
+### Test Env
+jaxlib=0.4. 35
+
+libtpu-nighly=20241028
```

Comment:
@suexu1025 Please correct typo libtpu-nighly > libtpu-nightly in this and other files.